### ISSUE TITLE
Feature request. Throttle lock widget.

### ISSUE THREAD

**Issue by Lefish63** [FrSkyRC/ETHOS-Feedback-Community#274]
Hi.

Really like Ethos and have been playing around with customizing screens with Ethos.
Could we have a throttle lock widget available when throttle lock or throttle cut is active for the screen please as a visual reference.
It has been a really nice feature on other systems.

Ta, Dave.

**Comment by bsongis-frsky** [comment]
As a widget displayed on Home Screen?

**Comment by Ceeb182** [comment]
One proposal :  

![Mt_On_Off h100px 127.58ppp](https://zupimages.net/up/21/20/r5hc.png)

**Comment by IJK-CZ** [comment]
In FrskyOS, there was a asign THR HOLD, so at least some similar. And Throttle hold should be available in all types of models (in FrOS it was not as option for Gliders...).

**Comment by Lefish63** [comment]
> As a widget displayed on Home Screen?

Yes that would be great, either a padlock image or something very obvious would be great.
Thanks Dave.

**Comment by IJK-CZ** [comment]
Icon or other symbol of THR lock in top status bar (visible on all screens) would be also fine.

**Comment by Lefish63** [comment]
> Icon or other symbol of THR lock in top status bar (visible on all screens) would be also fine.

Even better. 👍

**Comment by Hill2** [comment]
Perhaps also notified with voice?

like:
Throttle Armed
Throttle Not Armed

**Comment by Lefish63** [comment]
Yes that would be good, nice to have the sound as a default for the throttle armed not armed positions. 👍

**Comment by Hobby4life** [comment]
You can ofcourse create your own sounds.
For who likes it, I have created a soundpack with the default system sounds, and my extra added sound that includes all these throttle arm disarm, on off etc voices..
Als all template files are added to generate them with a voice you like.

It directly outputs in the correct wav format.
https://github.com/Hobby4life/ETHOS-OpenTX-Soundpacks-EN-NL

**Comment by mawzthefinn** [comment]
This would be very useful. Should be doable as a Lua though.

**Comment by Lefish63** [comment]
Indeed.
But as I have pointed out many times lots of people just like things simple, as in a lot of Frsky radio owners I look after and upgrade etc.
 A simple canned throttle lock option built in the radio firmware with default sounds satisfies the masses and is a safety feature well worth having. 

Just like the masses would like a canned mix for elevons but that's not there either.

**Comment by mawzthefinn** [comment]
The Tri-State Widget can be used for this with System Event-> Throttle Cut as the source.
